pkg/apis/vpc/v1alpha1/subnetconnectionbindingmap_types.go

```diff
@@ -0,0 +1,70 @@
+package v1alpha1
+
+import metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+// +kubebuilder:validation:XValidation:rule="!(has(self.targetSubnetSet) && has(self.targetSubnet))",message="Only one of targetSubnetSet or targetSubnet can be specified"
+type SubnetConnectionBindingMapSpec struct {
+	// SubnetName is the Subnet name which this SubnetConnectionBindingMap is associated.
+	// +kubebuilder:validation:XValidation:rule="self == oldSelf",message="subnetName is immutable"
+	SubnetName string `json:"subnetName"`
+	// TargetSubnetSetName specifies the target SubnetSet which a Subnet is bound to.
+	// +kubebuilder:validation:Optional
+	TargetSubnetSetName string `json:"targetSubnetSetName,omitempty"`
+	// TargetSubnetName specifies the target Subnet which a Subnet is bound to.
+	// +kubebuilder:validation:Optional
+	TargetSubnetName string `json:"targetSubnetName,omitempty"`
+	// VLANTrafficTag is the vlan tag configured in the binding. It can be empty, then the handler will choose a valid
+	// value based on the existing SubnetConnectionBindingMaps bound on the target Subnet or SubnetSet.
+	// +kubebuilder:validation:Maximum:=4095
+	// +kubebuilder:validation:Minimum:=1
+	// +kubebuilder:validation:Required
+	VLANTrafficTag int64 `json:"vlanTrafficTag"`
```

Comment:
Any reason to use int64, not int32?

Reply:
`int64` is used for VlanTrafficTag in the NSX model `SubnetConnectionBindingMap` in SDK. It is easier to keep consistent.